### PR TITLE
added hrm sport mode setting

### PR DESCRIPTION
**Pull request by nxdefiant**

### CODE REVIEW

File: apps/sportmode/settings.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  E.showMenu({
+    "" : { "title" : /*LANG*/"HRM sport mode" },
+    "< Back" : () => back(),
+    /*LANG*/'Sport mode': {
+      value: settings.mode,
+      min: -1, max: SPORT_MODES.length-1,
+      format: v => v === -1 ? /*LANG*/"Auto" : SPORT_MODES[v],
+      onchange: v => {
+        settings.mode = v;
+        writeSettings();
```
Comment (bobrippling) [inline apps/sportmode/settings.js:50]: If a user's got fastload enabled, this won't take effect until a full reload of the watch - perhaps we should set `hrmSportMode` here too?

Comment (nxdefiant) [reply apps/sportmode/settings.js:50]: Actually leaving settings does a reload

Comment (bobrippling) [reply apps/sportmode/settings.js:50]: So it does, nice!

Comment (nxdefiant) [reply apps/sportmode/settings.js:50]: But I'm thinking of adding this line anyway. What if we want to open this setting from e.g. heart rate monitor.

Comment (bobrippling) [reply apps/sportmode/settings.js:53]: Yeah that's a fair point, nice catch with the remove handler too

**Comment by nxdefiant** [comment]
should be ok now

**Comment by bobrippling** [comment]
Cool, thanks!

**Pull request merged**